ui/src/components/NavigationBar/NavBar.js

```diff
@@ -0,0 +1,250 @@
+import React, { useState } from "react";
+import {
+  Flex,
+  Center,
+  Stack,
+  Box,
+  Text,
+  Image,
+  Button,
+  Menu,
+  Container,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  Drawer,
+  DrawerBody,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerOverlay,
+  DrawerContent,
+  DrawerCloseButton,
+  useDisclosure,
+  Img,
+} from "@chakra-ui/react";
+import { Link } from "react-router-dom";
+import MainMenu from "./MainMenu";
+
+const NavBar = () => {
+  const [loggedIn, setLoggedIn] = useState(true);
+  const username = "name";
```

Comment:
shouldn't `setLoggedIn` properly check for the existence of the `user` object?
also, username is hardcoded, shouldn't be this way